lib/appmap/service/test_command_provider.rb

```diff
@@ -61,7 +59,7 @@ def minitest_commands
                 command: {
```

Comment:
At this point we are just “praying” that this will work because outside of Rails we can’t rely on much standardization. Hopefully the user has an opportunity to change/refine/control this command because it’s not too likely to work on its own.

Reply:
Right, they will be able to edit a command to run.